packages/contentful-app-extensions/disabled-fields/src/locations/Field.spec.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import Field from './Field';
+import { render } from '@testing-library/react';
+
+jest.mock('@contentful/react-apps-toolkit', () => ({
+  useSDK: () => ({
+    field: {
+      getValue: jest.fn(() => 'Hello World'),
+    },
+    window: {
+      startAutoResizer: jest.fn(),
+    },
+  }),
+}));
+
+describe('Field component', () => {
+  it('displays field text', () => {
+    const { getByText } = render(<Field />);
+
+    expect(getByText('Hello World')).toBeInTheDocument();
```

Comment:
wouldn't that be in the document even if it was a regular input? I don't think I care as much but in case you do maybe you should look for the <p>

Reply:
Yeah, I was not sure how to test that, it's just a simple paragraph that renders what it gets from `field.getValue`. In here `maybe you should look for the` what would be the suggestion? :)